lib/Runtime/Base/Constants.h

```diff
@@ -163,6 +163,10 @@ namespace Js
         static const size_t GlobalCodeLength        = _countof(_u("Global code")) - 1;
         static const size_t EvalCodeLength          = _countof(_u("eval code")) - 1;
         static const size_t UnknownScriptCodeLength = _countof(_u("Unknown script code")) - 1;
+
+        static const charcount_t NullStringLength = _countof(_u("Null")) - 1;
```

Comment:
The above lengths are all size_t - since you're touching this code anyway, can you update those to be charcount_t too?